src/Cooked/Attack/AddToken.hs

```diff
@@ -44,13 +44,11 @@ addTokenAttack extraTokens attacker = do
             map
               ( \(tName, amount) ->
                   let newMints = addToTxSkelMints (policy, redeemer, tName, amount) oldMints
-                      increment =
-                        txSkelMintsValue newMints
-                          <> ScriptutusTx.negate (txSkelMintsValue oldMints)
+                      increment = txSkelMintsValue newMints <> ScriptutusTx.negate (txSkelMintsValue oldMints)
```

Comment:
Was `ScriptutusTx` an intended name?

Reply:
absolutely not ! thanks for noticing I will change this.